src/posh-git.psm1

```diff
@@ -17,7 +17,7 @@ if ($psv.Major -lt 3 -and !$NoVersionWarn) {
 . $PSScriptRoot\GitUtils.ps1
 . $PSScriptRoot\GitPrompt.ps1
 . $PSScriptRoot\GitTabExpansion.ps1
-. $PSScriptRoot\ParamsExpansions.ps1
+. $PSScriptRoot\GitParamsExpansion.ps1
```

Comment:
Sorry for being a bit pedantic here, but could you make that `GitParamTabExpansion.ps1`.  Thanks.